### ISSUE TITLE
Notification close button fails contrast checks

### ISSUE THREAD

**Issue by Stanton** [jadu/pulsar#1250]
**Describe the bug**
The close button for notification does not meet AA contrast requirements.

**Screenshots**
![Screenshot 2020-06-03 at 10 42 25](https://user-images.githubusercontent.com/18653/83621799-0ddd5b00-a587-11ea-88d4-2baff47df452.png)
![Screenshot 2020-06-03 at 10 42 46](https://user-images.githubusercontent.com/18653/83621804-0fa71e80-a587-11ea-87e9-f4b12f59acaa.png)
